### ISSUE TITLE
Missing constraint when generating table columns for attributes that have minimum or maximum values specified.

### ISSUE THREAD

**Issue by Dariusms** [linkml/linkml#1924]
**Is your feature request related to a problem? Please describe.**
Currently one can specify a maximum_value and/or a minimum value for a slot or an attribute in LinkML. But when generating the ddl for the column it doesn't include that as a constraint.

**Describe the solution you'd like**
When minmum_value and/or maximum_value is specified for a slot or attribute, I would like to see the appropriate constraint created. 

**How important is this feature?** Select from the options below:
• Low - it's an enhancement but not crucial for work
• Medium - can do work without it; but it's important (e.g. to save time or for convenience)
• Important - it's a blocker and can't do work without it
Medium

**When will use cases depending on this become relevant?** Select from the options below:
• Short-term - 2-4 weeks
• Mid-term - 2-4 months
• Long-term - 6 months - 1 year
Mid-term

**Additional context**

**Comment by nlharris** [comment]
See also #1915

**Comment by Dariusms** [comment]
#1915 is about size this one is about value so  column can have a column of max size 32 chars the max value for an integer column might be 100 for a particular situation.

**Comment by nlharris** [comment]
Yes, I did not mean to suggest that they are redundant with each other; just that they are related.
